src/app/scroll-collapse/scroll-collapse.directive.ts

```diff
@@ -211,6 +223,7 @@ export class ScrollCollapseDirective implements AfterViewInit, OnDestroy {
   public calculateMinimiseMode(viewport: Viewport): void {
     this.minimiseMode =
       viewport.scrollY >= this.originalHeight + this.originalTop;
+    this.minimiseChange.emit(this.minimiseMode);
```

Comment:
should this only emit if there is a change?

Reply:
Updated